Sources/FluentKit/Query/Builder/QueryBuilder.swift

```diff
@@ -238,8 +243,46 @@ public final class QueryBuilder<Model>
             }
         }
         
+        let model = Model.init()
+        switch query.action {
+        case .delete:
+            if !self.forceDelete {
+                model.touchTimestamps(.delete, .update)
+                query.action = .update
+                query.input = [.dictionary(model.input.values)]
+            }
+        case .create:
+            var data: [DatabaseQuery.Value] = []
+            
+            for case .dictionary(var nested) in query.input {
+                if let createTimestamp = Model().timestamps.filter({ $0.trigger == .create }).first {
```

Comment:
There's no limit to how many `on: .create` timestamps a model can have so doing `first` here is not technically correct.